src/app/frontend/replicasetlist/replicasetlist.html

```diff
@@ -34,3 +34,9 @@
   </kd-content>
 </kd-content-card>
 <kd-zero-state ng-if="$ctrl.shouldShowZeroState()"></kd-zero-state>
+
+<kd-auto-refresh source="::$ctrl.replicaSetListResource"
```

Comment:
I can't reproduce it in my version. Maybe related to optionList bug which I've solved in my copy.